File: app/src/main/java/acr/browser/lightning/search/engine/GoogleSearch.java
package acr.browser.lightning.search.engine;

import acr.browser.lightning.R;
import acr.browser.lightning.constant.Constants;

/**
 * The Google search engine.
 * <p>
 * See https://www.google.com/images/srpr/logo11w.png for the icon.
 */
public class GoogleSearch extends BaseSearchEngine {

    public GoogleSearch() {
        super("file:///android_asset/google.png", Constants.GOOGLE_SEARCH, R.string.search_engine_google);
    }
}

File: app/src/main/java/acr/browser/lightning/search/engine/GoogleSearch.kt
package acr.browser.lightning.search.engine

import acr.browser.lightning.R

/**
 * The Google search engine.
 *
 * See https://www.google.com/images/srpr/logo11w.png for the icon.
 */
class GoogleSearch : BaseSearchEngine(
        "file:///android_asset/google.png",
        "https://www.google.com/search?client=lightning&ie=UTF-8&oe=UTF-8&q=",
        R.string.search_engine_google
)
